contracts/release1/build3/build-metadata-R1B3.json

```diff
@@ -0,0 +1,42 @@
+{
+  "ui": "",
+  "change": "Replaced the `STORE_PERMISSION` permission by the more specific `STORE_NUMBER_PERMISSION` and `STORE_ACCOUNT_PERMISSION` and added the `NumberStored` and `AccountStored` events.",
```

Comment:
I know this is just an example, but I would encourage developers to add human readable descriptions by giving an example ourselves

Reply:
Changed in 8d1269e32f811223b47a34c552eeddddeaa2a922